eng/build.yml

```diff
@@ -1,7 +1,12 @@
 steps:
 - task: UseDotNet@2
   inputs:
-    version: 2.2.402
+    version: 2.2.207
```

Comment:
we have some integration tests did with `netcoreapp2.1;netcoreapp2.2` and roll forward major is not supported by default https://github.com/dotnet/designs/blob/master/accepted/runtime-binding.md